Deleting_Content.md

```diff
@@ -0,0 +1,44 @@
+# Things Are Sometimes Better Unread
+
+In any large digitial community with a diverse group of humans, communication can break down. Whether it's unintential miscommunication or intentional boundary pushing, Administrators are occasionally requested to delete a message someone wrote in the public channels.
+
+# In Brief
+
+* Admins may delete messages that violate the Code of Conduct.
+* In non-trolling cases, the authors of those messages will be notified and may be given a chance to modify the message themselves.
```

Comment:
Is this possible with our 5 minute edit window?

Reply:
Good point Graham.

I've added some clarification around the 30 min edit window and that Admin assistance will be required after 30 min has lapsed to delete the message.